src/ai/backend/manager/models/scaling_group.py

```diff
@@ -315,6 +320,59 @@ class ScalingGroup(graphene.ObjectType):
     scheduler_opts = graphene.JSONString()
     use_host_network = graphene.Boolean()
 
+    # Dynamic fields.
+    agent_count_by_status = graphene.Field(
+        graphene.Int,
+        description="Added in 24.03.5.",
+        status=graphene.String(default_value="ALIVE"),
```

Comment:
```suggestion
        description=f"Added in 24.03.5. Should be one of {[s.name for s in AgentStatus]}",
        status=graphene.String(default_value=AgentStatus.ALIVE.name),
```
With such detail description, clients will be able to know what values are available for this field.
And `Enum` type values have `name` string field. Let's use it